client/src/App.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
```

Comment:
I wish I could come up with more comments but this all looks fine to me. I also learnt about UUID so thanks for that! You already know styling is incomplete. Good job!

Reply:
Yep, seconded from me. You have a deployed site that works. Alas, I'm not at a level to provide any useful code comments but well done!

Reply:
yeah UUID is really cool! lets you create unique keys which react likes to use to update components.